alpaca/data/mappings.py

```diff
@@ -39,6 +39,8 @@
     "minuteBar": "minute_bar",
     "dailyBar": "daily_bar",
     "prevDailyBar": "previous_daily_bar",
+    "impliedVolatility": "implied_volatility",
+    "greeks": "greeks",
```

Comment:
@impredicative thank you for the comment.
Yeah, it looks a confusing but. We are using this `SNAPSHOT_MAPPING` in the below [1] to create `mapped_snapshot` dict to convert `raw_data` dict into an object. Since we do not pass `raw_data` into an initializer of an object but mapped_snapshot  only, we need to map required fields into `mapped_snapshot` dict.

*1 https://github.com/alpacahq/alpaca-py/blob/eae556d8f1f12b57080c6efccac360c4ab02e807/alpaca/data/models/snapshots.py#L122-L126